src/utils/getFilterData.js

```diff
@@ -0,0 +1,49 @@
+export function sortData(products, sortBy, priceRange, sortByRating) {
+  switch (sortBy) {
+    case "CLEAR":
+      products = products;
+    case "LOW_TO_HIGH":
+      products = products.sort(
+        (a, b) => parseFloat(a.price) - parseFloat(b.price)
+      );
+      break;
+    case "HIGH_TO_LOW":
+      products = products.sort(
+        (a, b) => parseFloat(b.price) - parseFloat(a.price)
+      );
+      break;
+    default:
+      products = products;
+  }
+  if (sortByRating) {
+    products = products.filter((item) => item.rating >= sortByRating);
+  }
+  if (priceRange) {
+    products = products.filter((item) => item.price <= +priceRange);
+  }
+  return products;
+}
+
+const union = (...arr) => {
```

Comment:
Could have named this function better. I fail to understand what the union is for.

Reply:
I will update